src/sasl/cyrsasl_external_verification.erl

```diff
@@ -34,21 +34,28 @@
 -export([verify_creds/1]).
 
 -include_lib("jid/include/jid.hrl").
+-include("mongoose_logger.hrl").
 
 -spec verify_creds(Creds :: mongoose_credentials:t()) ->
     {ok, Username :: binary()} | {error, Error :: binary()}.
 
 verify_creds(Creds) ->
     AuthId = mongoose_credentials:get(Creds, auth_id, undefined),
-    XmppAddr = case mongoose_credentials:get(Creds, xmpp_addresses) of
+    Addrs = mongoose_credentials:get(Creds, xmpp_addresses),
+    XmppAddr = case Addrs of
```

Comment:
Why are this operation and the one below expanded in two lines? Don't think it adds any value to this PR 😕

Reply:
so we can put XmppAddr value into LOG_ERROR, for debugging problems with certificates :)